dhcp_server/DOCS.md

```diff
@@ -38,99 +38,126 @@ networks:
+
+A class-B network would have a netmask of `255.255.0.0`, covering addresses from `x.x.0.0` to `x.x.255.255` (65,534 usable addresses).
+
+If unsure, use `255.255.255.0`.
 
 #### Option: `networks.range_start`
 
-Defines the start IP address for the DHCP server to lease IPs for.
-Use this together with the `range_end` option to define the range of IP
-addresses the DHCP server operates in.
+The start IP address for the DHCP lease range. Used with `range_end` to define the range of IP's available for lease by the DHCP server.
 
 #### Option: `networks.range_end`
 
-Defines the end IP address for the DHCP server to lease IPs for.
+The last IP address for the range of IP addresses available for lease by the DHCP server.
 
 #### Option: `networks.broadcast`
 
-The broadcast address specific to the lease range. For example, if your
-IP addresses are `192.168.1.x`, the broadcast address is usually `192.168.1.255`.
+The broadcast IP address of the network, typically the first three octets of your class-C network plus `.255`, e.g., `192.168.1.255` for `192.168.1.0`.
 
 #### Option: `networks.gateway`
 
-Sets the gateway address for that the DHCP server hands out to its clients.
-This is usually the IP address of your router.
+The router (gateway) address where traffic is routed from your network onto the Internet.
+
+This is typically the first three octets of your class-C network plus `.1`, e.g., `192.168.1.1` for `192.168.1.0`.
+
+This is required. If this address is missing or wrong, traffic will not flow onto the Internet.
 
 #### Option: `networks.interface`
 
-The network interface to listen to for this network, e.g., `end0`.
+The network interface to listen on, e.g., `eth0`, the default for Home Assistant OS.
 
 ### Option: `hosts` (optional)
 
-This option defines settings for one or host definitions for the DHCP server.
+A list of hosts to be assigned fixed IP addresses. Requires the host's MAC address.
+
+Fixed IP addresses can be anywhere within the network, as long as they are not duplicates.  They are not required to be within the `range_start` and `range_end`. This range is for dynamically allocated IP addresses.
 
-It allows you to fix a host to a specific IP address.
+By default, no hosts are configured and none are required for DHCP server function.
 
-By default, non are configured.
+Example:
+```yaml
+hosts:
+  - name: MyRouter
+    mac: aa:bb:cc:dd:ee:ff
+    ip: 192.168.1.1
+    6: 94.140.14.15  # Option 6 w/Family-Friendly DNS Server (AdGuard)
+```
 
 #### Option: `hosts.name`
 
-The name of the hostname you'd like to fix an address for.
+The name of the host (client) to be assigned a fixed IP address.
 
 #### Option: `hosts.mac`
 
-The MAC address of the client device.
+The MAC address of the host (client) device.
 
 #### Option: `hosts.ip`
 
-The IP address you want the DHCP server to assign.
+The IP address assigned to the host (client) device.
+
+#### Option: Number between 1 - 254
```

Comment:
I don't think the Supervisor/Frontend allows to set these keys in the add-on config without adding them to the schema in config.yaml no?

What we could do here as a work around is add a config option `extra_dhcp_options` which is a list of itmes with a `key` and `value` property.